cloud/configs/nginx.conf

```diff
@@ -14,9 +14,13 @@ server {
     access_log /var/log/nginx/etagecom.io_access.log;
     error_log /var/log/nginx/etagecom.io_error.log;
 
+    #CANNOT redirect POST messages (they change to GETs). Reject instead.
+    if ($http_x_forwarded_proto != "https" & $request_method = POST) {
```

Comment:
I take my comment about 'looks good' back, the '&' operator seems to cause nginx problems, at least in my environment (nginx 1.4.6). Is this the syntax you tested?